src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx

```diff
@@ -332,6 +341,11 @@ export const getFormSchema = (
           "ui:description":
             "When unscheduled, tasks from previous revisions will be unscheduled when the equivalent task in a newer commit finishes successfully.",
         },
+        stepbackDisabled: {
+          "ui:widget": widgets.RadioBoxWidget,
+          "ui:description":
+            "Override all stepback settings for the project. Disabling stepback won't cancel any active stepback tasks, but will prevent any future ones.",
```

Comment:
"Disabling will override all enabled stepback settings", maybe?
(Enabling this doesn't mean that stepback is on for everything, just that we aren 't force disabling it, so we'd want to make that clear)